dwh/indicator_lib.py

```diff
@@ -115,6 +115,35 @@ def find_age_band(birth_date: str, end_date: datetime) -> str:
     return '25-49'
   return '50+'
 
+def find_time_buckets(recorded_date: str,) -> str:
```

Comment:
The if statements here are not necessary. Please convert datetime to `%Y-%m` to avoid boundaries issue when more than one is passed.